tests/helpers/Participant.ts

```diff
@@ -0,0 +1,238 @@
+
+            $('body').toggleClass('notransition');
+
+            document.title = `${name}`;
+
+            console.log(`lib-jitsi-meet version: ${JitsiMeetJS.version} sessionId: ${sessionId}`);
+        }, this._name, driver.sessionId));
+
+        // disable the blur effect in firefox as it has some performance issues
+        if (driver.capabilities.firefox) {
+            parallel.push(driver.execute(() => {
+                try {
+                    const blur = document.querySelector('.video_blurred_container');
+
+                    if (blur) {
+                        document.querySelector('.video_blurred_container').style.display = 'none';
+                    }
+                } catch (e) {} // eslint-disable-line no-empty
+            }));
+        }
+
+        if (skipInMeetingChecks) {
+            await Promise.all(parallel);
+
+            return;
+        }
+
+        parallel.push(this.waitForIceConnected());
+        parallel.push(this.waitForSendReceiveData());
+
+        await Promise.all(parallel);
+    }
+
+    /**
+     * Waits for the page to load.
+     */
+    async waitForPageToLoad() {
+        return this.driver.waitUntil(
+            () => this.driver.execute(() => document.readyState === 'complete'),
+            {
+                timeout: 30 * 1000, // 30 seconds
+                timeoutMsg: 'Timeout waiting for Page Load Request to complete.'
+            }
+        );
+    }
+
+    /**
+     * Waits to join the muc.
+     */
+    async waitToJoinMUC() {
+        return this.driver.waitUntil(
+            () => this.driver.execute(() => APP.conference.isJoined()),
+            {
+                timeout: 10 * 1000, // 10 seconds
+                timeoutMsg: 'Timeout waiting to join muc.'
+            }
+        );
+    }
+
+    /**
+     * Waits for ICE to get connected.
+     */
+    async waitForIceConnected() {
+        const driver = this.driver;
+
+        return driver.waitUntil(async () =>
+            driver.execute(() => APP.conference.getConnectionState() === 'connected'), {
+            timeout: 15000,
+            timeoutMsg: 'expected ICE to be connected for 15s'
+        });
+    }
+
+    /**
+     * Waits for send and receive data.
+     */
+    async waitForSendReceiveData() {
+        const driver = this.driver;
+
+        return driver.waitUntil(async () =>
+            driver.execute(() => {
+                try {
+                    const stats = APP.conference.getStats();
+                    const bitrateMap = stats.bitrate;
+                    const rtpStats = {
+                        uploadBitrate: bitrateMap?.upload,
+                        downloadBitrate: bitrateMap?.download
+                    };
+
+                    return rtpStats.uploadBitrate > 0 && rtpStats.downloadBitrate > 0;
+                } catch (e) {
+                    console.error('error', e.message, e.cause);
+                }
+            }), {
+            timeout: 15000,
```

Comment:
Elsewhere in the file you have XX * 1000, let's be consistent? Note that JS now allows to write this: `15_000` which is very legible.